modules/plugin/csv/src/main/java/org/geotools/data/csv/CSVFeatureSource.java

```diff
@@ -45,11 +45,12 @@ public CSVFeatureSource(ContentEntry entry, Query query) {
         super(entry, query);
     }
 
+    @Override
     public CSVDataStore getDataStore() {
         return (CSVDataStore) super.getDataStore();
     }
 
-    // docs start getBoundsInternal
```

Comment:
Is this just left over from the old tutorial code? Just want to make sure we aren't breaking a docs reference.

Reply:
The docs code is now completely broken out and I think I've finally found all of these.